ci-operator/step-registry/cert-manager/clusterissuer/OWNERS

```diff
@@ -0,0 +1,8 @@
+approvers:
```

Comment:
Could we change each sub-folder's OWNERS file to symbolic Link to `../OWNERS`.

Reply:
Thought of this. But both way is fine. And there is also others like https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift/e2e/test/qe/OWNERS slightly differs with https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift/e2e/test/qe/destructive/OWNERS . If no big issue, I prefer to keep as is without modification for faster merge.